.gitignore

```diff
@@ -1,4 +1,5 @@
 .idea
 .vscode
 gatus
-db.db
\ No newline at end of file
+db.db
+config/config.yml
```

Comment:
```suggestion
```

Reply:
Actually this entry was intentional, because the file does not exist in the repository and I created it for testing. But of course I can delete the line again if you want to.

Reply:
Fair enough